server/src/main/java/org/candlepin/util/X509V3ExtensionUtil.java

```diff
@@ -407,9 +412,29 @@ private Branding getBranding(Pool pool, String productId) {
             }
             content.setArches(archesList);
 
+            Boolean enabled = pc.getEnabled();
```

Comment:
I think there's a performance problem here, we're looking up the SKU product, checking for the attributes, and splitting the strings all within a for loop that runs once per content set. However none of that data ever changes. We should probably check both attributes before entering the loop and build up a set we can check for values in.